nubis/puppet/salesforce_sfmc.pp

```diff
@@ -0,0 +1,118 @@
+cron::daily { "${project_name}-salesforce_sfmc":
+  user    => 'etl',
+  command => "nubis-cron ${project_name}-salesforce_sfmc /opt/etl/salesforce_sfmc/run",
+}
+
+python::pyvenv { "${virtualenv_path}/sfmc-fetcher" :
+  ensure  => present,
+  version => '3.4',
+  require => [
+    File[$virtualenv_path],
+  ],
+}
+
+python::pyvenv { "${virtualenv_path}/vertica-csv-loader" :
```

Comment:
Puppet doesn't support duplicated, this isn't needed, as salesforce.pp already takes care of this.